x-pack/plugins/security_solution/public/entity_analytics/components/asset_criticality/asset_criticality_badge.tsx

```diff
@@ -7,23 +7,29 @@
 import React from 'react';
 import { EuiHealth } from '@elastic/eui';
 import { euiLightVars } from '@kbn/ui-theme';
-import { FormattedMessage } from '@kbn/i18n-react';
 import { CRITICALITY_LEVEL_TITLE } from './translations';
-import type { CriticalityLevel } from '../../../../common/entity_analytics/asset_criticality/types';
+import type { CriticalityLevelWithUnassigned } from '../../../../common/entity_analytics/asset_criticality/types';
 
-export const CRITICALITY_LEVEL_COLOR: Record<CriticalityLevel, string> = {
+export const CRITICALITY_LEVEL_COLOR: Record<CriticalityLevelWithUnassigned, string> = {
   extreme_impact: '#E7664C',
-  high_impact: '#D6BF57',
-  medium_impact: '#54B399',
-  low_impact: euiLightVars.euiColorMediumShade,
+  high_impact: '#DA8B45',
+  medium_impact: 'D6BF57',
+  low_impact: '#54B399',
+  unassigned: euiLightVars.euiColorMediumShade,
 };
 
 export const AssetCriticalityBadge: React.FC<{
-  criticalityLevel: CriticalityLevel;
+  criticalityLevel?: CriticalityLevelWithUnassigned;
   style?: React.CSSProperties;
   className?: string;
   dataTestSubj?: string;
-}> = ({ criticalityLevel, style, dataTestSubj = 'asset-criticality-badge', className }) => {
+}> = ({
+  criticalityLevel: maybeCriticalityLevel,
+  style,
+  dataTestSubj = 'asset-criticality-badge',
+  className,
+}) => {
+  const criticalityLevel = maybeCriticalityLevel ?? 'unassigned';
```

Comment:
Nitpick: maybe just do a default assignment? that way we can get rid of the extra `maybeCriticalLevel` variable

Reply:
actually, does that prop actually need to be optional? It wasn't before 🤔

Reply:
> Nitpick: maybe just do a default assignment? that way we can get rid of the extra maybeCriticalLevel variable

🤦 not sure why I didnt think of this

> actually, does that prop actually need to be optional? It wasn't before 🤔

We used to have a dedicated component for the component that allowed optional, now that we have an unassigned status I have combined the components into this one